src/prefect/_internal/pydantic/v2_schema.py

```diff
@@ -106,4 +107,8 @@ def create_v2_schema(
     # ensure backwards compatibility by copying $defs into definitions
     if "$defs" in schema:
         schema["definitions"] = schema["$defs"]
+
+    # Remove additional properties from schema
+    schema = remove_nested_keys(["additionalProperties"], schema)
```

Comment:
I'm thinking I need to fix this, as this is modifying the user schema

wondering if its related to this: https://github.com/pydantic/pydantic/pull/11644, but I would have thought 2.11.1 would resolve this

Reply:
removed in [90ee8af](https://github.com/PrefectHQ/prefect/pull/17698/commits/90ee8afb7abd7210b30e2eaf62ef3700f6f6a7e7)